src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/InteropServices/CSharpDynamicInterfaceCastableImplementation.Fixer.cs

```diff
@@ -0,0 +1,203 @@
+
+            SyntaxNode SealMemberDeclaration(SyntaxNode declaration, ISymbol symbol)
+            {
+                switch (symbol.Kind)
+                {
+                    case SymbolKind.Method:
+                        {
+                            DeclarationModifiers modifiers = generator.GetModifiers(declaration)
+                               .WithIsAbstract(false)
+                               .WithIsVirtual(false)
+                               .WithIsSealed(true);
+                            if (symbol.IsAbstract)
+                            {
+                                declaration = generator.WithStatements(declaration, defaultMethodBodyStatements);
+                            }
+                            return generator.WithModifiers(declaration, modifiers);
+                        }
+                    case SymbolKind.Property:
+                        {
+                            DeclarationModifiers modifiers = generator.GetModifiers(declaration)
+                               .WithIsAbstract(false)
+                               .WithIsVirtual(false)
+                               .WithIsSealed(true);
+                            if (symbol.IsAbstract)
+                            {
+                                IPropertySymbol prop = (IPropertySymbol)symbol;
+                                if (prop.GetMethod is not null)
+                                {
+                                    declaration = generator.WithGetAccessorStatements(declaration, defaultMethodBodyStatements);
+                                }
+                                if (prop.SetMethod is not null)
+                                {
+                                    declaration = generator.WithSetAccessorStatements(declaration, defaultMethodBodyStatements);
+                                }
+                            }
+                            return generator.WithModifiers(declaration, modifiers);
+                        }
+                    case SymbolKind.Event:
+                        {
+                            DeclarationModifiers modifiers = generator.GetModifiers(declaration)
+                               .WithIsAbstract(false)
+                               .WithIsVirtual(false)
+                               .WithIsSealed(true);
+                            if (symbol.IsAbstract)
+                            {
+                                declaration = GenerateEventImplementation((IEventSymbol)symbol, declaration, generator, defaultMethodBodyStatements);
+                            }
+                            return generator.WithModifiers(declaration, modifiers);
+                        }
+                }
+
+                return declaration;
+            }
+        }
+
+        protected override bool CodeFixSupportsDeclaration(SyntaxNode declaration)
+        {
+            return !declaration.IsKind(SyntaxKind.VariableDeclarator);
```

Comment:
Any test shows this returning false (i.e, a variable declarator)?

Reply:
This is tested by `DynamicInterfaceCastableImplementation_Events_MultipleInOneDeclaration_NoBody_CS_Diagnostic`